site/content/features/typepatterns.md

```diff
@@ -1,12 +1,11 @@
 ---
 title: Pattern Matching for switch (JEP 427)
 copyright: Cay S. Horstmann 2022. All rights reserved.
-jep: 427
-jdkversion: 19
+jep: 441
+jdkversion: 21
 ---
 
-
-Pattern matching for switch expressions and statements appeared as a preview feature in Java 17 ([JEP 406](https://openjdk.java.net/jeps/406)) and Java 18 ([JEP 420](https://openjdk.java.net/jeps/420)). This article covers the third round of preview in Java 19 ([JEP 427](https://openjdk.java.net/jeps/427)). The feature is mostly straightforward, with a few sharp edges. At the end of each section is a “sandbox” with somewhat contrived code to try out the syntax variations.
+Pattern matching for switch expressions and statements appeared as a preview feature in Java 17 ([JEP 406](https://openjdk.java.net/jeps/406)), Java 18 ([JEP 420](https://openjdk.java.net/jeps/420)), Java 19 ([JEP 427](https://openjdk.java.net/jeps/427)), and Java 20 ([JEP 433](https://openjdk.java.net/jeps/433)) This article covers the final specification which is part of Java 21 ([JEP 441](https://openjdk.java.net/jeps/441)). The feature is mostly straightforward, with a few sharp edges. At the end of each section is a “sandbox” with somewhat contrived code to try out the syntax variations.
```

Comment:
Missing period before? Probably it should be:

> and Java 20 ([JEP 433](https://openjdk.java.net/jeps/433)). This article covers

Reply:
Fixing right now